### DISCUSSION TITLE
MHonArc repo

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1615] [Team Posts]
@ldidry , which branch we'd be better to target in our pr, `master` or `develop`?
(cf. [network](https://github.com/sympa-community/MHonArc/network))

**Comment by ldidry** [comment]
I suggest to use this:
- all PR are done to `develop`
- once we are happy of what is in `develop`, we merge `develop` in `master`, create a tag and release a new version

This way, the default branch, `master`, is always at the last version.

It’s the [gitflow workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow), a widely used git workflow..

**Comment by racke** [comment]
It's up to you, but the even more common workflow is to target master. I don't think we need the extra complexity of a develop branch. BTW: thanks a lot for your patches.
